deployment/volcano/config/kube-batch-ci.conf

```diff
@@ -0,0 +1,11 @@
+actions: "reclaim, allocate, backfill, preempt"
```

Comment:
I thought you were going to remove them in this file, not the original file. And if I am right, you need to update testcase as well.

Reply:
Since in CI, we need to test every action, deployment used in CI is same as the deployment provided for user.  So, I have added new config file for CI which will be loaded only during CI run, when user is deploying using helm it will use default config file which will have allocate, backfill alone, which is out requirement as mentioned in the issue https://github.com/kubernetes-sigs/volcano/issues/770